src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <li
+                key={tab.id}
+                className={classNames({ 'is-active': activeTab === tab })}
```

Comment:
You should not compare objects directly. Instead, compare them by their unique identifier. In this case, you should compare `tab.id` with `activeTab.id`. This is a common mistake that can lead to unexpected bugs.